tree/ntuple/v7/src/RColumnElement.hxx

```diff
@@ -1,38 +1,20 @@
-/// \file ROOT/RColumnElement.hxx
-/// \ingroup NTuple ROOT7
-/// \author Jakob Blomer <jblomer@cern.ch>
-/// \date 2018-10-09
-/// \warning This is part of the ROOT 7 prototype! It will change without notice. It might trigger earthquakes. Feedback
-/// is welcome!
-
-/*************************************************************************
- * Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.               *
- * All rights reserved.                                                  *
- *                                                                       *
- * For the licensing terms see $ROOTSYS/LICENSE.                         *
- * For the list of contributors see $ROOTSYS/README/CREDITS.             *
- *************************************************************************/
-
-#ifndef ROOT7_RColumnElement
-#define ROOT7_RColumnElement
-
-#include <ROOT/RConfig.hxx>
-#include <ROOT/RError.hxx>
-#include <ROOT/RFloat16.hxx>
+// RColumnElement concrete implementations
+//
+// Note that this file is in the src directory and not in the inc directory because we need the ability
+// to override R__LITTLE_ENDIAN for testing purposes.
+// This is not a particularly clean or correct solution, as the tests that do this will end up with two different
+// definitions of some RColumnElements, so we might want to change this mechanism in the future. In any case, these
+// definitions are implementation details and should not be exposed to a public interface.
+
+#include <ROOT/RColumnElementBase.hxx>
+#include <ROOT/EExecutionPolicy.hxx>
```

Comment:
Why do we need this include?

Reply:
Not sure, it was added automatically by my editor. That's an annoying functionality that I need to disable because it's so easy to miss such things.